packages/react-integration/cypress/integration/dropdown.spec.ts

```diff
@@ -35,23 +27,6 @@ describe('Dropdown Demo Test', () => {
     cy.get('#dropdown').should('not.have.class', 'pf-m-expanded');
   });
 
-  it('Verify dropdown menu items disabled', () => {
```

Comment:
Don't we want to keep this to test the interaction?

Reply:
This interaction is invalid. You cannot click on an item with `pointer-events: none` and cypress now enforces this.